### ISSUE TITLE
Public link does not display colors I set using the site

### ISSUE THREAD

**Issue by owen-dall** [mapbox/geojson.io#485]
http://bl.ocks.org/d/52b6cbc7e5c17337ab7a

See screen shot of how it looks on geojson.io.
![buildings with color](https://cloud.githubusercontent.com/assets/10927/11117274/6f2275d2-8906-11e5-8cf9-2a51d61f063c.png)

**Comment by tmcw** [comment]
Indeed, for some reason we aren't using L.mapbox.featureLayer in the embed.
